api_testsuite.go

```diff
@@ -245,6 +254,22 @@ func (ats *Suite) parseAndRunTest(v any, manifestDir, testFilePath string, k, re
 		return false
 	}
 
+	// Parse as template always
+	requestBytes, lErr := loader.Render(testObj, filepath.Join(manifestDir, dir), nil)
+	if lErr != nil {
+		r.SaveToReportLog(lErr.Error())
+		logrus.Error(fmt.Errorf("can not render template (%s): %s", testFilePath, lErr))
+		return false
+	}
+
+	// If objects are different, we did have a Go template, recurse one level deep
+	if string(requestBytes) != string(testObj) {
+		return ats.parseAndRunTest([]byte(requestBytes), filepath.Join(manifestDir, dir),
+			testFilePath, k, parallelRepititions, isParallelPathSpec, r, loader)
+	}
```

Comment:
Why is that important? Why do we dive one level deep? imho this is not necessary...